### PR TITLE
[ADD]Add access to table of field 'Divers/Miscellaneous'

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

**Review by eilst — APPROVED** [review]

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<odoo>
+    <record id="action_miscellaneous" model="ir.actions.act_window">
+        <field name="name">Miscellaneous</field>
+        <field name="res_model">resource.calendar.miscellaneous</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem id="configuration_sub_menu"
+              name="Configuration"
+              sequence="350"
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: Why sequence is 350?

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+              action="action_miscellaneous"
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: no action

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+              parent="calendar.mail_menu_calendar"
+              groups="group_resource_calendar_manager"/>
+    <menuitem id="miscellaneous_sub_menu"
+              name="Miscellaneous"
+              parent="configuration_sub_menu"
+              action="action_miscellaneous"
+              sequence="20"
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: Why you need to add sequence?

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="action_miscellaneous" model="ir.actions.act_window">
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: better action_resource_calendar_miscellaneous

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+              action="action_miscellaneous"
+              parent="calendar.mail_menu_calendar"
+              groups="group_resource_calendar_manager"/>
+    <menuitem id="miscellaneous_sub_menu"
+              name="Miscellaneous"
+              parent="configuration_sub_menu"
+              action="action_miscellaneous"
+              sequence="20"
+              groups="group_resource_calendar_manager"/>
+    <record id="miscellaneous_tree" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: better resource_calendar_miscellaneous_tree (same comment for form)

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        </field>
+    </record>
+    <record id="miscellaneous_form" model="ir.ui.view">
+        <field name="name">Miscellaneous Form</field>
+        <field name="model">resource.calendar.miscellaneous</field>
+        <field name="arch" type="xml">
+            <form string="Miscellaneous">
+                <sheet>
+                    <div>
+                        <h1>
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: add group to display the label name

File: project_resource_calendar/views/room_type_view.xml [hunk 7]
```diff
@@ -6,12 +6,19 @@
         <field name="view_type">form</field>
         <field name="view_mode">tree,form</field>
     </record>
+    <menuitem id="configuration_sub_menu"
```
Comment (rimbendhaou) [inline project_resource_calendar/views/room_type_view.xml:9]: Why 2 menuitem with the same id?

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 4 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 5 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 6 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 2 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 3 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/miscellaneous_view.xml (hunk 1 above)
Comment (jananjoy) [reply project_resource_calendar/views/miscellaneous_view.xml]: @rimbendhaou Changed

File: project_resource_calendar/views/room_type_view.xml (hunk 7 above)
Comment (jananjoy) [reply project_resource_calendar/views/room_type_view.xml:9]: @rimbendhaou Changed

File: project_resource_calendar/views/room_type_view.xml [hunk 8]
```diff
@@ -6,11 +6,14 @@
...
```
Comment (rimbendhaou) [inline project_resource_calendar/views/room_type_view.xml:9]: The last menu in the calendar root menu is  Reports it has a sequence of 300. In order to put the Configuration menu after we can use sequence=310. So later if we want to add new menus between them we can add sequence 301,302,303,304,...309 (so i think better to add sequence=310)

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_resource_calendar/views/room_type_view.xml (hunk 8 above)
Comment (jananjoy) [reply project_resource_calendar/views/room_type_view.xml:9]: @rimbendhaou Changed

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**